src/lib/util.ts

```diff
@@ -0,0 +1,18 @@
+declare let globalThis: any;
+
+// From https://www.npmjs.com/package/btoa
+const ployfills = {
+  btoa: function (str: string | Buffer) {
+    let buffer;
+
+    if (str instanceof Buffer) {
+      buffer = str;
+    } else {
+      buffer = Buffer.from(str.toString(), "binary");
```

Comment:
Is it safe to use this? Is this going to require Buffer in the browser now? Or will this be tree-shaken out?

Reply:
Just making sure we understand the implications here? Is this going to be an issue with our browser builds? I don't want to bundle Buffer if we don't have to (or aren't already).

Reply:
My understanding: We won't be bundling Buffer, but expecting it to already be in the global environment in which this code is called. In modern contexts in which a btoa polyfil is needed (node environments), Buffer will already be built in. In environments where Buffer isn't built in (browsers), this line of code will never be hit because btoa is built in.

Reply:
Well the `if (str instance Buffer)` line will be hit every time this runs, and in my browser if I run that line, I see:
```
> "" instanceof Buffer
< ReferenceError: Can't find variable: Buffer
```

So this seems unsafe to me.

Reply:
Unless `ployfills` is only _included_ when we need it? In which case, that seems fine!